### PR TITLE
Report properly whether pod shares host network

### PR DESCRIPTION
**Pull request by rhatdan**
Fixes: https://github.com/containers/podman/issues/14028

Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
@Luap99 PTAL

File: libpod/pod.go [hunk 1]
```diff
@@ -159,6 +159,27 @@ func (p *Pod) CPUQuota() int64 {
 	return 0
 }
 
+// NetMode returns the Network mode given by the user ex: pod, private...
+func (p *Pod) NetMode() string {
+	infra, err := p.runtime.GetContainer(p.state.InfraContainerID)
+	if err != nil {
+		return ""
+	}
+	ctrSpec := infra.config.Spec
```
Comment (Luap99) [inline libpod/pod.go:168]: this should just call infra.NetworkMode()

File: libpod/pod.go [hunk 2]
```diff
@@ -159,6 +159,27 @@ func (p *Pod) CPUQuota() int64 {
 	return 0
 }
 
+// NetMode returns the Network mode given by the user ex: pod, private...
+func (p *Pod) NetMode() string {
```
Comment (Luap99) [inline libpod/pod.go:163]: call this NetworkMode() to be consistent with the container function.

**Review by Luap99 — COMMENTED** [review]
LGTM

**Pull request merged**